src/info/updates/archive.md

```diff
@@ -8,11 +8,22 @@ meta:
 
 # Archived updates from the FAC
 
-The FAC is committed to working transparently and openly. In addition to our regular updates, you can find historical updates below.
-
-## Week of October 30, 2023
-
-We launched search and are ready for users to begin searching single audit reports. This initial iteration of search leverages six different filters for sorting data.
+The FAC is committed to working transparently and openly. In addition to <a href=https://www.fac.gov/info/updates/>our regular updates</a>, you can find historical updates below.
```

Comment:
```suggestion
The FAC is committed to working transparently and openly. In addition to [our regular updates](https://www.fac.gov/info/updates/), you can find historical updates below.
```

It looks like it rendered correctly, but this anchor is missing some quotes (`<a href="site.com">text</a>`). We could also use markdown syntax as in the suggestion above! 

edit: looks like there are a bunch of these. Rather than comment on all of them, I might just make a commit to this branch if that's ok.

Reply:
I did some cleanup in da99cf8dee1f8e09f2ad272e821464c18783bc41 to:

1. Add `""` quotes to html anchor tags.
2. Convert markdown links to [markdown style](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links).
3. Remove stray `<p></p>` paragraph tags. The [USWDS Accordion code examples](https://designsystem.digital.gov/components/accordion/) all show accordions with `<p>sample text</p>` wrapped in paragraph tags, but our site generator does this for us when we supply Markdown text. When it converts Markdown into HTML, it wraps blocks of text in `<p></p>` tags, lists in `<ul></ul>` tags, headings in `<h3></h3>` tags, etc, and plops those elements inside the `<div class="usa-accordion__content"></div>` tags you've provided.